### PR TITLE
Template extra volumes in helm chart

### PR DESCRIPTION
**Pull request by nirroz93**
same as https://github.com/apache/airflow/pull/29357

---
Same as https://github.com/apache/airflow/pull/29357, seems like it was missed
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by hussein-awala — CHANGES_REQUESTED** [review]
in the PR you mention, the values file was updated by adding an example for each extra volumes field and adding a comment before the example:
```
It can be templated like in the following example:
```
You need to do the same thing for `gitSync`, `dagProcessor`, `flower`, `createUserJob` and `pgbouncer`?

**Review by jedcunningham — APPROVED** [review]
Sorry for the delay here @nirroz93. This looks good. Can you resolve the conflict so I can get this merged? Thanks!

**Comment by nirroz93** [comment]
@jedcunningham - done

File: chart/values.yaml [hunk 1]
```diff
@@ -2146,9 +2201,14 @@ dags:
     #  runAsUser: 65533
     #  runAsGroup: 0
 
+
     securityContexts:
       container: {}
```
Comment (jedcunningham) [inline chart/values.yaml]:
```suggestion
    securityContexts:
      container: {}

```

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
@hussein-awala, want to take another look?

**Review by hussein-awala — APPROVED** [review]
Looks good

**Comment by jedcunningham** [comment]
Looks like static checks and one test are failing. I'd suggest that you [set up the pre-commit hooks](https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#pre-commit-hooks), they help catch the static check issues early 👍.

**Comment by eladkal** [comment]
@nirroz93  can you address the failing static tests?

**Comment by nirroz93** [comment]
Yes, I will look at it later (sorry for disappearing)

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
Thanks @nirroz93! Congrats on your first commit 🎉

**Pull request merged**